hello-pepr-schedule/tsconfig.json

```
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "esModuleInterop": true,
+    "lib": [
+      "ES2022"
+    ],
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    "outDir": "dist",
+    "resolveJsonModule": true,
+    "rootDir": ".",
+    "strict": false,
+    "target": "ES2022",
+    "useUnknownInCatchVariables": false
+  },
+  "include": [
+    "**/*.ts"
+  ]
```

```suggestion
  ],
  "exclude": [
    "**/*.test.*",
    "./dist/**/*",
    "./dev/**/*"
  ]
```

We're using some exclude patterns like this in the other modules just so we don't have to worry about TS errors in utils / dist / test helper code -- should probably mirror that here too, no?